Octokit/Http/CustomAwaiter.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+namespace Octokit
+{
+    public static class CustomAwaiter
+    {
+        public static TaskAwaiter<IReadOnlyList<T>> GetAwaiter<T>(this IDeferredRequest<T> request)
```

Comment:
Still a breaking change. The API itself has changed from hot to cold and existing code now needs to .ToTask()

Right?